scripts/automation/install.sh

```diff
@@ -56,7 +56,7 @@ fi
 # Download the Cascade project from github
 
 CASCADE_HOME=/ihme/code/dismod_at
-CASCADE_DEVELOP_DIR=${CASCADE_HOME}/cascade_develop
+CASCADE_DEVELOP_DIR=${CASCADE_HOME}/clone_for_install
```

Comment:
Why change the name?

Reply:
Alec is on vacation, so this makes a new directory in order to avoid a directory with bad permissions. Will need to delete the other directory after he returns.